files/en-us/web/http/headers/content-security-policy/frame-ancestors/index.html

```diff
@@ -64,7 +64,7 @@ <h3 id="Sources">Sources</h3>
  </ul>
 
  <div class="notecard warning">
- <p>If no URL scheme is specified for a <code>host-source</code> and the iframe is loaded from an <code>https</code> URL, the URL for the page loading the iframe must also be <code>https</code>, per the W3C spec on <a href="https://w3c.github.io/webappsec-csp/2/#match-source-expression">matching source expressions</a>.</p>
+ <p>If no URL scheme is specified for a <code>host-source</code> and the iframe is loaded from an <code>https</code> URL, the URL for the page loading the iframe must also be <code>https</code>, per the W3C spec on <a href="https://www.w3.org/TR/CSP2/#match-source-expression">matching source expressions</a>.</p>
```

Comment:
```suggestion
 <p>If no URL scheme is specified for a <code>host-source</code> and the iframe is loaded from an <code>https</code> URL, the URL for the page loading the iframe must also be <code>https</code>, per the <a href="https://w3c.github.io/webappsec-csp/#match-url-to-source-expression">Does url match expression in origin with redirect count?</a> section of the CSP spec.</p>
```
The CSP2 is outdated, we should anyway change this instead to the relevant part of the current CSP spec.